rx-usecases-test/src/main/java/com/thefuntasty/mvvm/rxusecases/test/CompletableUseCase.kt

```diff
@@ -0,0 +1,76 @@
+package com.thefuntasty.mvvm.rxusecases.test
```

Comment:
What about change file name to `CompletableUseCaseMocking` or `CompletableUseCaseMock`?